### DISCUSSION TITLE
Allow more capitalization of letters in player names

### DISCUSSION THREAD

**Discussion by Plan414** [chromiecraft/chromiecraft#3479] [Suggestions]
The title explains it. I have no idea if this is even possible. 

The name "Plan" can only be spelled this way when selecting this name in game. 
Could it be changed to accept "pLan" ?
or even with numbers? "pLan414" ?

**Comment by BeefSteaker** [comment]
I think this is a limitation of the client. Not sure if they could do it even if they wanted to
